<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_user_question</class>
 <widget class="QDialog" name="Show_user_question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:grey;
}
QPushButton
{
	font: 75 14pt &quot;Cambria&quot;;
	background-color: #3de6af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	padding: 13px;
	border-radius:12px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Question">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QLabel" name="op1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="op2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="op3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="op5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
;
font: 75 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>651</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Question_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="op1_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="op2_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="op3_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="op4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
color:black;
border-radius:13px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Calibri&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string>*Enter The Number Between 1 To 4.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
